### PR TITLE
Searchbar style fix

### PR DESCRIPTION
**Pull request by jurgenizer**
A styling fix to remove the white area behind the CupertinoTextField. 
The fix makes search bar look like default iOS search bar.

before:
image1

after:
image0

### CODE REVIEW

File: veggieseasons/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -374,6 +374,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 8;
+				DEVELOPMENT_TEAM = "";
```
Comment (RedBrogdon) [inline veggieseasons/ios/Runner.xcodeproj/project.pbxproj]: These lines can just be taken out.

File: veggieseasons/lib/widgets/search_bar.dart [hunk 2]
```diff
@@ -39,6 +39,12 @@ class SearchBar extends StatelessWidget {
               child: CupertinoTextField(
                 controller: controller,
                 focusNode: focusNode,
+                 decoration: BoxDecoration( // to remove white area behind text field
```
Comment (RedBrogdon) [inline veggieseasons/lib/widgets/search_bar.dart]: @LongCatIsLooong/@xster, what's your thought here?

Is this a flaw in how the original code is using CupertinoTextField?

**Comment by RedBrogdon** [comment]
Ah, it looks like our automated tests have flagged the formatting in `search_bar.dart`. It looks like one of the new lines isn't indented correctly. Just run `flutter format` on that file and it'll be corrected.

File: veggieseasons/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jurgenizer) [reply veggieseasons/ios/Runner.xcodeproj/project.pbxproj]: lines removed

File: veggieseasons/lib/widgets/search_bar.dart [hunk 3]
```diff
@@ -39,6 +39,11 @@ class SearchBar extends StatelessWidget {
               child: CupertinoTextField(
                 controller: controller,
                 focusNode: focusNode,
+                decoration: BoxDecoration(
```
Comment (LongCatIsLooong) [inline veggieseasons/lib/widgets/search_bar.dart]: nit: [`CupertinoTextField`'s documentation](https://master-api.flutter.dev/flutter/cupertino/CupertinoTextField-class.html) states that "If you set the decoration property to null, the decoration will be removed entirely".  Maybe we should just set it to `null`?

Comment (jurgenizer) [reply veggieseasons/lib/widgets/search_bar.dart]: @LongCatIsLooong I agree. I have set the decoration property to null. I was not sure how to do it so I just did `decoration: BoxDecoration(),`

File: veggieseasons/lib/widgets/search_bar.dart [hunk 4]
```diff
@@ -39,6 +39,7 @@ class SearchBar extends StatelessWidget {
               child: CupertinoTextField(
                 controller: controller,
                 focusNode: focusNode,
+                decoration: BoxDecoration(),
```
Comment (LongCatIsLooong) [inline veggieseasons/lib/widgets/search_bar.dart]: According to the documentation `decoration` is nullable. So:
```suggestion
                decoration: null,
```

Comment (jurgenizer) [reply veggieseasons/lib/widgets/search_bar.dart]: Thanks! I updated it now

**Comment by RedBrogdon** [comment]
Awesome. Sounds like we've got the right change. @jurgenizer, I think there are still some changes in the project file that need to be taken out. Other than that, though, this looks ready to land!

**Pull request merged**